core/arch/arm32/plat-stm/system_config.in

```diff
@@ -40,6 +40,7 @@ endif
 # Common values
 
 CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= 1
+CFG_TEE_CORE_NO_TA_TRACE ?= n
```

Comment:
Since this is always used with !defined(CFG_TEE_CORE_NO_TA_TRACE) I suggest the other way around:
`CFG_TEE_CORE_TA_TRACE ?= y`.